templates/style.php

```diff
@@ -1,3 +1,5 @@
+@import "https://s1.wp.com/i/fonts/merriweather/merriweather.css?v=20160212";
```

Comment:
Thanks for looking @mjangda -- can we link to the fonts using an `@import` like this? If not, I can include the actual `@font-face` rules from merriweather.css here.